### ISSUE TITLE
Site responsiveness

### ISSUE THREAD

**Issue by saekort** [saekort/PFS-Scenariotracker#18]
Medium resolutions tend to get two rows of menus at the top, for example a ipad when flipped to have the long side be vertical. This needs attention.

The layout also needs some love on the extra small resolutions.

**Comment by FlavienKnuchel** [comment]
It is maybe not really the best place to discuss it, but this issues made me think about something: I was thinking about proposing an alternative design, swiss made, for the tracker. Then I said myself "that's a terrible idea, there's already a bootstrap implemented that goes well". 

But what about I propose an alternate design using bootstrap grid system and classes,  add an option inside the "other" menu that let you choose which design you want, implement the logic to handle this, and save the user choice in localstorage? Would it be usefull? Or my inner designer i having one of his crappy ideas again? :)

**Comment by saekort** [comment]
I would disagree with letting people choose between designs. It is a nightmare to support more then one.

That said, I am always open for more design ideas. How about you throw a quick mockup together with Pencil or Photoshop and attach it here so we can take a look?
